src/vm/amd64/jithelpers_fast.S

```diff
@@ -110,7 +110,7 @@ LEAF_ENTRY JIT_WriteBarrier, _TEXT
         movabs  rax, 0xF0F0F0F0F0F0F0F0
 
         // Touch the card table entry, if not already dirty.
-        shr     rdi, 0Bh
+        shr     rdi, 0x0B
```

Comment:
Interesting, so now only one syntax for hex values is allowed in the clang assembler? I am just curious, what was the error message for the original syntax?